tests/menubar-editor/navigate-to-menubar-in-reading-mode.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Navigation to menubar using reading mode</title>
+<link rel="author" title="Valerie Young" href="mailto:valerie@bocoup.com">
+<link rel="help" href="https://w3c.github.io/aria-practices/examples/menubar/menubar-2/menubar-2.html">
+<link rel="help" href="https://w3c.github.io/aria/#menubar">
+
+<script type="module">
+  import { verifyATBehavior, displayTestPageAndInstructions } from "../resources/aria-at-harness.mjs";
+
+  verifyATBehavior({
+    setupTestPage: function setupTestPage(testPageDocument) {

```

Comment:
As you can see here, the scripts expect "document", so we should replace "testPageDocument" with document.